tox.ini

```diff
@@ -2,47 +2,34 @@
 envlist = linters, py3, insync, integration
 isolated_build = True
 
-[tox:.package]
-## Required for poetry
-# note tox will use the same python version as under what tox is installed to package
-# so unless this is python 3 you can require a given python version for the packaging
-# environment via the basepython key
-basepython = python3
-
 [testenv]
-skip_install = true
-deps = poetry >= 1.0.5
+usedevelop = True
+setenv =
+  VIRTUAL_ENV={envdir}
+deps =
+    pytest
 commands =
-    poetry install
-    poetry env info
-    poetry run py.test -v test -k "not integration" {posargs}
+    py.test -v test -k "not integration" {posargs}
 
 [testenv:linters]
-basepython = python3
+deps =
+    flake8
+    pylint
+    yamllint
```

Comment:
what if we put this into test-requirements.txt ?

Then we'd have
```
[testenv]
basepython = python3
install_command = pip install {opts} {packages}
deps = -r{toxinidir}/requirements.txt
       -r{toxinidir}/test-requirements.txt
```

Reply:
I wanted to be super explicit here because I wanted to avoid pushing a broken package to pypi. 

yamllint declares a dependency on pyyaml. 

Right now, if you remove pyyaml from requirements.txt, the unit tests will fail. I like it this way.